src/main/java/com/bobocode/svydovets/bibernate/util/EntityUtils.java

```diff
@@ -0,0 +1,41 @@
+package com.bobocode.svydovets.bibernate.util;
+
+import com.bobocode.svydovets.bibernate.annotation.Entity;
+import com.bobocode.svydovets.bibernate.exception.EntityValidationException;
+import java.util.Arrays;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EntityUtils {
+    private EntityUtils() {}
+
+    public static <T> void validateEntity(Class<T> type) {
+        log.info("Validation %s".formatted(type.getName()));
```

Comment:
I beliave logger supports string formating with {} for instance `log.info("Validation {}", type.getName());`